extensions/wikia/SiteWideMessages/SpecialSiteWideMessages_body.php

```diff
@@ -1031,7 +1031,7 @@ static function getAllUserMessagesId($user, $filter_seen = true) {
 			. ' WHERE msg_removed = ' . MSG_REMOVED_NO
 			. ' AND msg_mode = ' . MSG_MODE_ALL
 			. ' AND (msg_expire IS NULL OR msg_expire > ' . $DB->AddQuotes(date('Y-m-d H:i:s')) . ')'
-			. " AND msg_date > '{$user->mRegistration}'"	//fix for ticket #2624
+			. " AND msg_date > '".$user->getRegistration()."'"	//fix for ticket #2624
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)